src/Store/TitleStoreConnector.re

```diff
@@ -113,6 +113,11 @@ let start = setTitle => {
       }
     );
 
+  let internalMaximizeEffect = () =>
+    Isolinear.Effect.createWithDispatch(~name="maximize", _dispatch =>
+      maximize()
+    );
```

Comment:
Minor nit. This doesn't need to be a function. You can also use just `create` instead of `createWithDispatch` 

```suggestion
  let internalMaximizeEffect =
    Isolinear.Effect.create(~name="maximize", () =>
      maximize()
    );
```

Reply:
I fixed this in the latest commit. I'll have to browse the Isolinear source sometime today to understand it better. Thanks!